src/main/cpp/mechanisms/Thing1Mech/decoratormods/Thing1Mech.h

```diff
@@ -28,7 +29,8 @@
 #include "robotstate/IRobotStateChangeSubscriber.h"
 #include "robotstate/RobotStateChanges.h"
 
-class Thing1Mech : public Thing1MechGen, public IRobotStateChangeSubscriber
+class Thing1Mech : public Thing1MechGen,
+				   public IRobotStateChangeSubscriber
```

Comment:
It would be nice to have this either all on one line or line up the public of each base class